server/src/controller/itemController.js

```diff
@@ -0,0 +1,15 @@
+import { itemModel } from "../models/itemModel.js";
+
+export const itemController = async (req, res) => {
+  //console.log(req.params);
+
+  const id = Number(req.params.categoryId);
+  // console.log(typeof id, "id");
+  try {
+    const item = await itemModel(id);
+    res.send(item);
+  } catch (e) {
+    //console.log(e);
+    return res.status(400).json({ e });
+  }
+};
```

Comment:
This will eventually need to be extended to control other item crud actions like create, delete, and update. You might want to restructure it now so it's clear this is only a get action. Maybe eventually it could look like this:

```js
export const getItem = () => {}
export const createItem = () => {}
export const deleteItem = () => {}
export const updateItem = () => {}
```